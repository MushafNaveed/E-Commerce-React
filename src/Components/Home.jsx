import React from "react";
import img1 from '../images/couch.png';
export default function Home() {
  return (
    <section id="Hero">
      <div className="container">
        <div className="row">
          <div className="col-md-6 mt-5 pt-5">
            <h1 className="mt-5 pt-5">Modern Interior Design Studio</h1>
            <p className="py-3">
              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Delectus
              ut dolorem repellat doloribus assumenda laboriosam numquam ullam
              fugit aperiam eaque!
            </p>
            <button className="btn btn-primary btn1 mx-3 rounded-pill">Show Now</button>
            <button className="btn btn-primary btn2 mx-3 rounded-pill">Explore</button>
          </div>
          <div className="col-md-6 mt-5 pt-5">
            <div className="Hero-imgwrap">
            <img src={img1} alt="" class="img-fluid"/>
            </div>
          </div>
        </div>
      </div>
    </section>
  );
}
