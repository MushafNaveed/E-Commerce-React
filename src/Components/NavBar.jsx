import React from 'react';
import svg1 from '../images/user.svg';
import { Link } from 'react-router-dom';
import Home from './Home';

export default function NavBar(props) {
  return (
    <section id='Header'>
        <div className="container-fluid">
            <div className="row">
                <div className="col-md-12">
                    <nav className='navbar navbar-expand-lg sticky-top'>
                        <a href="/" className='navbar-brand me-auto'>{props.Title}</a>
                        <ul className='navbar-nav navbarLinks me-5'>
                            <li className="nav-item"><a className='nav-link' href='#Hero'>Home</a></li>
                            <li className="nav-item"><a className='nav-link' href='#Explore'>Shop</a></li>
                            <li className="nav-item"><a className='nav-link' href='#about'>About Us</a></li>
                            <li className="nav-item"><a className='nav-link' href='#help'>Services</a></li>
                            <li className="nav-item"><a className='nav-link' href='#product'>Products</a></li>
                            <li className="nav-item"><a className='nav-link' href='#blog'>Blog</a></li>
                            <li className="nav-item"><a className='nav-link' href='#footer'>Contact Us</a></li>
                        </ul>
                        <ul class="custom-navbar-cta navbar-nav mb-2 mb-md-0 ms-5">
						<li><a className="nav-link" href="/"><img src={svg1}/></a></li>
					</ul>
                    </nav>
                </div>
            </div>
        </div>
    </section>
  );
}


