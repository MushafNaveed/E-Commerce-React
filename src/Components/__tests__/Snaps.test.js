import { render } from "@testing-library/react";
import NavBar from "../NavBar";
import Home from "../Home";
import Foot from "../Foot";

describe('SnapShot testing', () => {
  it('NavBar', () => {
    const NavBarComp= render(<NavBar/>);
    expect(NavBarComp).toMatchSnapshot();
  });
  it('Home', () => {
    const HomeComp= render(<Home/>);
    expect(HomeComp).toMatchSnapshot();
  });
  it('Foot', () => {
    const FootComp= render(<Foot/>);
    expect(FootComp).toMatchSnapshot();
  });
});
