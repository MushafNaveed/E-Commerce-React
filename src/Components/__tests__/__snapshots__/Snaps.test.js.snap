// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapShot testing Foot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="footer-section"
        id="footer"
      >
        <div
          class="container relative"
        >
          <div
            class="sofa-img mt-5"
          >
            <img
              alt="Image"
              class="img-fluid mt-5 pt-5"
              src="sofa.png"
            />
          </div>
          <div
            class="row"
          >
            <div
              class="col-lg-8"
            >
              <div
                class="subscription-form mt-5"
              >
                <h3
                  class="d-flex align-items-center"
                >
                  <span
                    class="me-1"
                  >
                    <img
                      alt="Image"
                      class="img-fluid"
                      src="envelope-outline.svg"
                    />
                  </span>
                  <span>
                    Subscribe to Newsletter
                  </span>
                </h3>
                <form
                  action="#"
                  class="row g-3"
                >
                  <div
                    class="col-auto"
                  >
                    <input
                      class="form-control"
                      placeholder="Enter your name"
                      type="text"
                    />
                  </div>
                  <div
                    class="col-auto"
                  >
                    <input
                      class="form-control"
                      placeholder="Enter your email"
                      type="email"
                    />
                  </div>
                  <div
                    class="col-auto"
                  >
                    <button
                      class="btn btn-primary"
                    >
                      <span
                        class="fa fa-paper-plane"
                      />
                    </button>
                  </div>
                </form>
              </div>
            </div>
          </div>
          <div
            class="row g-5 mb-5"
          >
            <div
              class="col-lg-4 mt-5 pt-5"
            >
              <div
                class="mb-4 footer-logo-wrap"
              >
                <a
                  class="footer-logo"
                  href="#"
                >
                  Brand
                  <span>
                    .
                  </span>
                </a>
              </div>
              <p
                class="mb-4"
              >
                Donec facilisis quam ut purus rutrum lobortis. Donec vitae odio quis nisl dapibus malesuada. Nullam ac aliquet velit. Aliquam vulputate velit imperdiet dolor tempor tristique. Pellentesque habitant
              </p>
              <ul
                class="list-unstyled custom-social"
              >
                <li>
                  <a
                    href="#"
                  >
                    <span
                      class="fa fa-brands fa-facebook-f"
                    />
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    <span
                      class="fa fa-brands fa-twitter"
                    />
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    <span
                      class="fa fa-brands fa-instagram"
                    />
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    <span
                      class="fa fa-brands fa-linkedin"
                    />
                  </a>
                </li>
              </ul>
            </div>
            <div
              class="col-lg-8 mt-5 pt-5"
            >
              <div
                class="row links-wrap mt-5"
              >
                <div
                  class="col-6 col-sm-6 col-md-3"
                >
                  <ul
                    class="list-unstyled"
                  >
                    <li>
                      <a
                        href="#"
                      >
                        About us
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Services
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Blog
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Contact us
                      </a>
                    </li>
                  </ul>
                </div>
                <div
                  class="col-6 col-sm-6 col-md-3"
                >
                  <ul
                    class="list-unstyled"
                  >
                    <li>
                      <a
                        href="#"
                      >
                        Support
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Knowledge base
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Live chat
                      </a>
                    </li>
                  </ul>
                </div>
                <div
                  class="col-6 col-sm-6 col-md-3"
                >
                  <ul
                    class="list-unstyled"
                  >
                    <li>
                      <a
                        href="#"
                      >
                        Jobs
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Our team
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Leadership
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Privacy Policy
                      </a>
                    </li>
                  </ul>
                </div>
                <div
                  class="col-6 col-sm-6 col-md-3"
                >
                  <ul
                    class="list-unstyled"
                  >
                    <li>
                      <a
                        href="#"
                      >
                        Nordic Chair
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Kruzo Aero
                      </a>
                    </li>
                    <li>
                      <a
                        href="#"
                      >
                        Ergonomic Chair
                      </a>
                    </li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
          <div
            class="border-top copyright"
          >
            <div
              class="row pt-4"
            >
              <div
                class="col-lg-6"
              >
                <p
                  class="mb-2 text-center text-lg-start"
                >
                  Copyright © 2024. All Rights Reserved. — Designed By MN.
                </p>
              </div>
              <div
                class="col-lg-6 text-center text-lg-end"
              >
                <ul
                  class="list-unstyled d-inline-flex ms-auto"
                >
                  <li
                    class="me-4"
                  >
                    <a
                      href="#"
                    >
                      Terms & Conditions
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Privacy Policy
                    </a>
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="footer-section"
      id="footer"
    >
      <div
        class="container relative"
      >
        <div
          class="sofa-img mt-5"
        >
          <img
            alt="Image"
            class="img-fluid mt-5 pt-5"
            src="sofa.png"
          />
        </div>
        <div
          class="row"
        >
          <div
            class="col-lg-8"
          >
            <div
              class="subscription-form mt-5"
            >
              <h3
                class="d-flex align-items-center"
              >
                <span
                  class="me-1"
                >
                  <img
                    alt="Image"
                    class="img-fluid"
                    src="envelope-outline.svg"
                  />
                </span>
                <span>
                  Subscribe to Newsletter
                </span>
              </h3>
              <form
                action="#"
                class="row g-3"
              >
                <div
                  class="col-auto"
                >
                  <input
                    class="form-control"
                    placeholder="Enter your name"
                    type="text"
                  />
                </div>
                <div
                  class="col-auto"
                >
                  <input
                    class="form-control"
                    placeholder="Enter your email"
                    type="email"
                  />
                </div>
                <div
                  class="col-auto"
                >
                  <button
                    class="btn btn-primary"
                  >
                    <span
                      class="fa fa-paper-plane"
                    />
                  </button>
                </div>
              </form>
            </div>
          </div>
        </div>
        <div
          class="row g-5 mb-5"
        >
          <div
            class="col-lg-4 mt-5 pt-5"
          >
            <div
              class="mb-4 footer-logo-wrap"
            >
              <a
                class="footer-logo"
                href="#"
              >
                Brand
                <span>
                  .
                </span>
              </a>
            </div>
            <p
              class="mb-4"
            >
              Donec facilisis quam ut purus rutrum lobortis. Donec vitae odio quis nisl dapibus malesuada. Nullam ac aliquet velit. Aliquam vulputate velit imperdiet dolor tempor tristique. Pellentesque habitant
            </p>
            <ul
              class="list-unstyled custom-social"
            >
              <li>
                <a
                  href="#"
                >
                  <span
                    class="fa fa-brands fa-facebook-f"
                  />
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  <span
                    class="fa fa-brands fa-twitter"
                  />
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  <span
                    class="fa fa-brands fa-instagram"
                  />
                </a>
              </li>
              <li>
                <a
                  href="#"
                >
                  <span
                    class="fa fa-brands fa-linkedin"
                  />
                </a>
              </li>
            </ul>
          </div>
          <div
            class="col-lg-8 mt-5 pt-5"
          >
            <div
              class="row links-wrap mt-5"
            >
              <div
                class="col-6 col-sm-6 col-md-3"
              >
                <ul
                  class="list-unstyled"
                >
                  <li>
                    <a
                      href="#"
                    >
                      About us
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Services
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Blog
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Contact us
                    </a>
                  </li>
                </ul>
              </div>
              <div
                class="col-6 col-sm-6 col-md-3"
              >
                <ul
                  class="list-unstyled"
                >
                  <li>
                    <a
                      href="#"
                    >
                      Support
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Knowledge base
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Live chat
                    </a>
                  </li>
                </ul>
              </div>
              <div
                class="col-6 col-sm-6 col-md-3"
              >
                <ul
                  class="list-unstyled"
                >
                  <li>
                    <a
                      href="#"
                    >
                      Jobs
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Our team
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Leadership
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Privacy Policy
                    </a>
                  </li>
                </ul>
              </div>
              <div
                class="col-6 col-sm-6 col-md-3"
              >
                <ul
                  class="list-unstyled"
                >
                  <li>
                    <a
                      href="#"
                    >
                      Nordic Chair
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Kruzo Aero
                    </a>
                  </li>
                  <li>
                    <a
                      href="#"
                    >
                      Ergonomic Chair
                    </a>
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
        <div
          class="border-top copyright"
        >
          <div
            class="row pt-4"
          >
            <div
              class="col-lg-6"
            >
              <p
                class="mb-2 text-center text-lg-start"
              >
                Copyright © 2024. All Rights Reserved. — Designed By MN.
              </p>
            </div>
            <div
              class="col-lg-6 text-center text-lg-end"
            >
              <ul
                class="list-unstyled d-inline-flex ms-auto"
              >
                <li
                  class="me-4"
                >
                  <a
                    href="#"
                  >
                    Terms & Conditions
                  </a>
                </li>
                <li>
                  <a
                    href="#"
                  >
                    Privacy Policy
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`SnapShot testing Home 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        id="Hero"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-md-6 mt-5 pt-5"
            >
              <h1
                class="mt-5 pt-5"
              >
                Modern Interior Design Studio
              </h1>
              <p
                class="py-3"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Delectus ut dolorem repellat doloribus assumenda laboriosam numquam ullam fugit aperiam eaque!
              </p>
              <button
                class="btn btn-primary btn1 mx-3 rounded-pill"
              >
                Show Now
              </button>
              <button
                class="btn btn-primary btn2 mx-3 rounded-pill"
              >
                Explore
              </button>
            </div>
            <div
              class="col-md-6 mt-5 pt-5"
            >
              <div
                class="Hero-imgwrap"
              >
                <img
                  alt=""
                  class="img-fluid"
                  src="couch.png"
                />
              </div>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      id="Hero"
    >
      <div
        class="container"
      >
        <div
          class="row"
        >
          <div
            class="col-md-6 mt-5 pt-5"
          >
            <h1
              class="mt-5 pt-5"
            >
              Modern Interior Design Studio
            </h1>
            <p
              class="py-3"
            >
              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Delectus ut dolorem repellat doloribus assumenda laboriosam numquam ullam fugit aperiam eaque!
            </p>
            <button
              class="btn btn-primary btn1 mx-3 rounded-pill"
            >
              Show Now
            </button>
            <button
              class="btn btn-primary btn2 mx-3 rounded-pill"
            >
              Explore
            </button>
          </div>
          <div
            class="col-md-6 mt-5 pt-5"
          >
            <div
              class="Hero-imgwrap"
            >
              <img
                alt=""
                class="img-fluid"
                src="couch.png"
              />
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`SnapShot testing NavBar 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        id="Header"
      >
        <div
          class="container-fluid"
        >
          <div
            class="row"
          >
            <div
              class="col-md-12"
            >
              <nav
                class="navbar navbar-expand-lg sticky-top"
              >
                <a
                  class="navbar-brand me-auto"
                  href="/"
                />
                <ul
                  class="navbar-nav navbarLinks me-5"
                >
                  <li
                    class="nav-item"
                  >
                    <a
                      class="nav-link"
                      href="#Hero"
                    >
                      Home
                    </a>
                  </li>
                  <li
                    class="nav-item"
                  >
                    <a
                      class="nav-link"
                      href="#Explore"
                    >
                      Shop
                    </a>
                  </li>
                  <li
                    class="nav-item"
                  >
                    <a
                      class="nav-link"
                      href="#about"
                    >
                      About Us
                    </a>
                  </li>
                  <li
                    class="nav-item"
                  >
                    <a
                      class="nav-link"
                      href="#help"
                    >
                      Services
                    </a>
                  </li>
                  <li
                    class="nav-item"
                  >
                    <a
                      class="nav-link"
                      href="#product"
                    >
                      Products
                    </a>
                  </li>
                  <li
                    class="nav-item"
                  >
                    <a
                      class="nav-link"
                      href="#blog"
                    >
                      Blog
                    </a>
                  </li>
                  <li
                    class="nav-item"
                  >
                    <a
                      class="nav-link"
                      href="#footer"
                    >
                      Contact Us
                    </a>
                  </li>
                </ul>
                <ul
                  class="custom-navbar-cta navbar-nav mb-2 mb-md-0 ms-5"
                >
                  <li>
                    <a
                      class="nav-link"
                      href="/"
                    >
                      <img
                        src="user.svg"
                      />
                    </a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      id="Header"
    >
      <div
        class="container-fluid"
      >
        <div
          class="row"
        >
          <div
            class="col-md-12"
          >
            <nav
              class="navbar navbar-expand-lg sticky-top"
            >
              <a
                class="navbar-brand me-auto"
                href="/"
              />
              <ul
                class="navbar-nav navbarLinks me-5"
              >
                <li
                  class="nav-item"
                >
                  <a
                    class="nav-link"
                    href="#Hero"
                  >
                    Home
                  </a>
                </li>
                <li
                  class="nav-item"
                >
                  <a
                    class="nav-link"
                    href="#Explore"
                  >
                    Shop
                  </a>
                </li>
                <li
                  class="nav-item"
                >
                  <a
                    class="nav-link"
                    href="#about"
                  >
                    About Us
                  </a>
                </li>
                <li
                  class="nav-item"
                >
                  <a
                    class="nav-link"
                    href="#help"
                  >
                    Services
                  </a>
                </li>
                <li
                  class="nav-item"
                >
                  <a
                    class="nav-link"
                    href="#product"
                  >
                    Products
                  </a>
                </li>
                <li
                  class="nav-item"
                >
                  <a
                    class="nav-link"
                    href="#blog"
                  >
                    Blog
                  </a>
                </li>
                <li
                  class="nav-item"
                >
                  <a
                    class="nav-link"
                    href="#footer"
                  >
                    Contact Us
                  </a>
                </li>
              </ul>
              <ul
                class="custom-navbar-cta navbar-nav mb-2 mb-md-0 ms-5"
              >
                <li>
                  <a
                    class="nav-link"
                    href="/"
                  >
                    <img
                      src="user.svg"
                    />
                  </a>
                </li>
              </ul>
            </nav>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
